services/s3/src/main/java/software/amazon/awssdk/services/s3/multipart/S3MultipartExecutionAttribute.java

```diff
@@ -16,11 +16,14 @@
 package software.amazon.awssdk.services.s3.multipart;
 
 import software.amazon.awssdk.annotations.SdkProtectedApi;
+import software.amazon.awssdk.core.async.listener.PublisherListener;
 import software.amazon.awssdk.core.interceptor.ExecutionAttribute;
 import software.amazon.awssdk.core.interceptor.SdkExecutionAttribute;
 
 @SdkProtectedApi
-public final class S3PauseResumeExecutionAttribute extends SdkExecutionAttribute {
+public final class S3MultipartExecutionAttribute extends SdkExecutionAttribute {
```

Comment:
We generally should not make breaking changes in protected APIs, but since this is a new class we recently added and I doubt anyone is using it, it's probably fine.